src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js

```diff
@@ -149,7 +151,9 @@ class CommentForm extends React.Component<Props, State> {
                             onChange={this.onMentionSelectorChangeHandler}
                             onFocus={onFocus}
                             onMention={getMentionWithQuery}
-                            placeholder={tagged_message ? undefined : formatMessage(messages.commentWrite)}
+                            placeholder={
+                                tagged_message ? undefined : placeholder ?? formatMessage(messages.commentWrite)
```

Comment:
I think we should have a test for this logic.

Reply:
Agree! I'll add it tomorrow morning